apisix/plugins/ext-plugin/init.lua

```diff
@@ -288,7 +288,14 @@ local function handle_extra_info(ctx, input)
 
     local res
     local info_type = req:InfoType()
-    if info_type == extra_info.Var then
+    if info_type == extra_info.RespBody + 1 then  --4 set_body
```

Comment:
Could you add the new field in https://github.com/api7/ext-plugin-proto?
You can read https://github.com/apache/apisix/blob/master/docs/en/latest/internal/plugin-runner.md for the implemented detail.